spec/features/feedback_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'Feedback', type: :feature do
+  before(:each) { visit feedback_url }
+  
+  it 'Submit feedback' do
+    @feedback = Feedback.new(:name => "Joe Public", :email => "foo@example.com",
+      :comment => "I can't submit a petition for some reason", :petition_link_or_title => 'link')
```

Comment:
This should and can be moved into a let like so `let(:feedback)  { Feedback.new(name: 'Joe Public', email: 'foo@example.com', comment: 'I can't submit a petition for some reason', petition_link_or_title: 'link') }`.